Makefile

```diff
@@ -60,7 +72,7 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build --pull --no-cache . -t ${IMG}
+	docker buildx build --pull --no-cache --platform=$(PLATFORMS) $(DESTARG) . -t ${IMG}
```

Comment:
Cloud we add a new rule for this command?

```
docker-buildx:
    docker buildx build --pull --no-cache --platform=$(PLATFORMS) $(DESTARG) . -t ${IMG}
```

Reply:
Sure, makes sense. I added it as docker-multiarch, so it wouldn't need changing if Docker changes buildx in the future.